Mathlib/Data/Complex/Module.lean

```diff
@@ -55,31 +55,33 @@ attribute [local ext] Complex.ext
 -- Test that the `SMul ℚ ℂ` instance is correct.
 example : (Complex.instSMulRealComplex : SMul ℚ ℂ) = (Algebra.toSMul : SMul ℚ ℂ) := rfl
 
-instance [SMul R ℝ] [SMul S ℝ] [SMulCommClass R S ℝ] : SMulCommClass R S ℂ where
+instance (priority := 90) [SMul R ℝ] [SMul S ℝ] [SMulCommClass R S ℝ] : SMulCommClass R S ℂ where
```

Comment:
Can you add a comment about these priorities to the module docstring? Something noting the general heuristic and the fact that benchmarking favored it.

Reply:
I previously made #11692 to have single identifier for such changes. One could easily imagine a future where all these need to be found.